aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -0,0 +1,51 @@
+/// This module provides an interface for aggregators (version 2).
+/// Only skeleton - for AggregagtorSnapshot - is provided at this time,
+/// to allow transition of usages.
+module aptos_framework::aggregator_v2 {
+    use std::string::String;
+
+    /// The generic type supplied to the aggregator snapshot is not supported
+    const EUNSUPPORTED_AGGREGATOR_SNAPSHOT_TYPE: u64 = 5;
+
+    struct AggregatorSnapshot<Element> has store, drop {
+        value: Element,
+    }
+
+    public native fun create_snapshot<Element: copy + drop>(value: Element): AggregatorSnapshot<Element>;
+
+    public native fun copy_snapshot<Element: copy + drop>(snapshot: &AggregatorSnapshot<Element>): AggregatorSnapshot<Element>;
+
+    public native fun read_snapshot<Element>(snapshot: &AggregatorSnapshot<Element>): Element;
+
+    public native fun string_concat<Element>(before: String, snapshot: &AggregatorSnapshot<Element>, after: String): AggregatorSnapshot<String>;
+
+    #[test]
+    public fun test_correct_read() {
```

Comment:
Do you need to enable the feature for the tests?

And let's add a test that without enabling feature, we cannot create a snapshot?

Reply:
Done.